sr_robot_launch/launch/sr_hardware_control_loop.launch

```diff
@@ -168,22 +172,15 @@
   <!-- Trajectory mode -->
   <group if="$(arg arm_ctrl)">
     <group if="$(arg arm_trajectory)">
-      <group if="$(arg hand_ctrl)">
```

Comment:
Why the if was eliminated?

Reply:
@beatrizleon  deleted cause as you can see, no matter what hand_ctrl value is, it will do exactly the same thing. The if/else statement here is redundant.